1 dimensional kalman filter (x axis) using odometry and laserscans

ohilho opened the pull request:
see demo.launch

ohilho committed:
1 dimensional kalman filter (x axis) using odometry and laserscans

ohilho committed:
add comments

ohilho committed:
add comments

ryul1206 reviewed (changes requested):
형 실례지만, 혹시 이 코드는 로봇이 있어야 쓸 수 있는 코드인가요?
아니면 로봇이 없어도 쓸 수 있는 코드인가요?
만약 후자에 해당하면 root/README에 이런 식으로 추가해 주실 수 있으신가요?
```markdown
- turtle_control (폴더명)
  - [@ohilho](https://github.com/ohilho)
  - turtle_control (turtle_control.cpp)
    - `roslaunch ~~~~` (실행방법)
  - turtle_localization (turtle_localization.cpp)
    - `roslaunch ~~~~` (실행방법)
```

Commit:
Add x_odom to plot

1) Delay rosbag play on launch for waiting rqt_plot initializing.
2) Add x_odom for plot.
3) Set initial x as first measurement of sensor.
4) Fix minor bug.

Commit:
Update README.md for `turtle_control`

ryul1206 committed:
Merge pull request #3 from rise-lab-skku/feature/tutle_control/update

Feature/tutle control/update

ryul1206 committed:
Merge branch 'main' into feature/turtle_control
